src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -154,18 +168,29 @@ other | Bus | Bus | Bus | Bus
 
 
 ### lines.txt
-`lines` are provided in the nodes **CompositeFrame/frames/ServiceFrame/lines/Line**. There could be multiple `ServiceFrame` to read.
+`lines` are provided in the nodes **CompositeFrame/frames/ServiceFrame[]/lines/Line[]**. The only `ServiceFrame` containing the lines has an id property set to `STIF:CODIFLIGNE:ServiceFrame:lineid`.
 
 NTFS field | Netex-IDFM element | Mapping rule/Comment
 --- | --- | ---
-line_id | *Line/@id* | This field is prefixed. 
-network_id | *Network/@id* | id of the network containing a reference to this line in `Network/members/LineRef/@ref`. This field is prefixed. 
-line_code | *Line/ShortName* | 
-line_name | *Line/Name* | 
+line_id | Line/@id | This field is prefixed. 
+network_id | Network/@id | id of the network containing a reference to this line in `Network/members/LineRef/@ref`. This field is prefixed. 
```

Comment:
These are the old specifications. Network is now defined by Line/RepresentedByGroupRef/@ref, line color and text color are absent, ...

Reply:
thanks, it's fixed